release/cloud_publish.template.json

```diff
@@ -1,12 +1,12 @@
 {
   "steps": [
     {
-      "name": "gcr.io/istio-io/istio-builder:0.5.7",
+      "name": "gcr.io/istio-testing/istio-builder:0.5.7",
```

Comment:
are we retiring the use of istio-io project?

Reply:
no, we are not, istio-builder is being pushed to istio-testing as part of our normal update for this docker image. We frequently forget to push to istio-io, also some other release scripts use istio-testing container. Reducing confusion by using a single source.